components/basejob/Basejobform.vue

```diff
@@ -194,14 +203,23 @@ export default {
       validator: (value) => [0, 10].includes(value),
     },
   },
+  components: {
+    'vue-form-json-schema': VueFormJsonSchema,
+  },
   data() {
     return {
       allEmailEvents: 0,
       isOpen: 0,
+      state: {},
+      schema: baseJobSchema,
     }
   },
   computed: {
     ...mapMultiRowFields(['sbatch']),
+    ...mapFields({ sbatchSchema: 'sbatchXnat2Bids' }),
+    uiSchema() {
+      return baseJobUISchema
+    },
```

Comment:
since `baseJobUISchema` is a constant, should this be `data` instead of `computed`? or will it someday be a more computedy thing?

Reply:
Good catch. Moved to data!